lib/run/index.js

```diff
@@ -71,6 +71,14 @@ module.exports = function (options, callback) {
         if (!_.isObject(options.collection)) {
             return callback(new Error('newman: expecting a collection to run'));
         }
+        // sanitize environment option
+        if (!options.environment) {
```

Comment:
Question - we do this here or in config module? Essentially config is supposed to sanitise it before it gets here

The fact that we are discussing this makes me think that we should organise it more evidently.